docs/DeveloperGuide.md

```diff
@@ -533,6 +534,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **Hard disk**: data storage device in laptops and computers.
 * **.JSON file**: JavaScript Object Notation, stores and transports data using a human-readable text format.
 * **SocialBook**: SocialBook is an address book hence SocialBook and AddressBook are interchangable. SocialBook is used to reference the product while AddressBook refers to the Class object in source code.
+* **Social worker**: Social work refers to full-time employees of the [Ministry of Social and Family Developement (MSF) or its subsidaries](https://www.msf.gov.sg)
```

Comment:
Is this meant to be something like:
Social worker: full-time employees of ....

As is right now, it doesn't look very clear to me whether this is a definition of social worker or social work.

Reply:
oops, typo. meant to be `social worker`